skorch/callbacks/logging.py

```diff
@@ -747,32 +756,30 @@ class SacredLogger(Callback):
 
     To use this logger, you first have to install Sacred:
 
-    $ python -m pip install sacred
+    ``$ python -m pip install sacred``
 
-    You might also install pymongo to use a mongodb backend. See the upstream_
-    documentation for more details. Once you have installed it, you can set up
-    a simple experiment and pass this Logger as a callback to your skorch
-    estimator:
+    You might also install pymongo to use a mongodb backend. See the `upstream
+    documentation <https://github.com/IDSIA/sacred#installing>`_ for more
+    details. Once you have installed it, you can set up a simple experiment and
+    pass this Logger as a callback to your skorch estimator:
 
-    # contents of sacred-experiment.py
+    Examples
+    --------
+    >>> # contents of sacred-experiment.py
     >>> import numpy as np
     >>> from sacred import Experiment
     >>> from sklearn.datasets import make_classification
     >>> from skorch.callbacks.logging import SacredLogger
     >>> from skorch.callbacks.scoring import EpochScoring
     >>> from skorch import NeuralNetClassifier
     >>> from skorch.toy import make_classifier
-
```

Comment:
One way to split this up is to add a comment describing the code.

XREF: https://numpydoc.readthedocs.io/en/latest/format.html#examples

Reply:
Yes, but I wanted to avoid that. My goal would be to just have a block of code with some blank lines:

```python
from skorch import NeuralNetClassifier
from skorch.toy import make_classifier

ex = Experiment()

@ex.config
def my_config():
    max_epochs = 20
    lr = 0.01

X, y = make_classification()
```
 
Adding comments seems unnecessary to me in this case.

Reply:
Ah I see. I do not think there is a great way to do this.